### PR TITLE
Bump github.com/containers/{storage,image,buildah}

### PR DESCRIPTION
**Pull request by nalind**
Bump github.com/containers/storage from v1.12.13 to v1.13.2, github.com/containers/image from v2.0.0 to v3.0.2 (and its github.com/vbauerster/mpb requirement from v3.3.4 to v3.4.0), github.com/containers/buildah from v1.9.1 to v1.10.1, and pin k8s.io/apimachinery, k8s.io/cli-runtime, and openshift/api to the versions used by OpenShift 4.2.

### CODE REVIEW

File: glide.yaml [hunk 1]
```diff
@@ -4,17 +4,21 @@ import:
...
 - package: k8s.io/kubernetes
   repo:    https://github.com/openshift/kubernetes.git
   version: origin-4.2-kubernetes-1.14.0
+- package: k8s.io/cli-runtime
+  repo:    https://github.com/openshift/kubernetes-cli-runtime.git
+  version: origin-4.2-kubernetes-1.14.0
 # 2. openshift
 - package: github.com/openshift/api
-  version: master
+  version: release-4.2
```
Comment (adambkaplan) [inline glide.yaml]: If you're switching api to `release-4.2`, switch `client-go` as well.

File: glide.yaml [hunk 2]
```diff
@@ -4,17 +4,21 @@ import:
 - package: k8s.io/api
   version: kubernetes-1.14.0
 - package: k8s.io/apimachinery
-  version: kubernetes-1.14.0
+  repo:    https://github.com/openshift/kubernetes-apimachinery.git
+  version: origin-4.2-kubernetes-1.14.0
```
Comment (adambkaplan) [inline glide.yaml]: Is there a reason why we need to use our fork? Is there a carry patch there that we need?

File: glide.yaml (hunk 1 above)
Comment (nalind) [reply glide.yaml]: Ah, didn't check for that.  Changing it.

File: glide.yaml (hunk 2 above)
Comment (nalind) [reply glide.yaml]: We started needing `k8s.io/apimachinery/pkg/apis/meta/v1beta1.AddToScheme()` in `k8s.io/kubernetes/pkg/kubectl/scheme.init()` (in https://github.com/openshift/kubernetes.git's origin-4.2-kubernetes-1.14.0 branch).

File: glide.yaml (hunk 1 above)
Comment (nalind) [reply glide.yaml]: Ah, `openshift/client-go` isn't `k8s.io/client-go`.  Pinning it.

**Comment by adambkaplan** [comment]
/retitle Bug 1754695: Bump containers/{storage,image,buildah}

**Pull request merged**